cmd/server_test.go

```diff
@@ -7,8 +7,63 @@ import (
 
 	"github.com/rancher/cli/config"
 	"github.com/stretchr/testify/assert"
+	"github.com/urfave/cli"
 )
 
+func TestServerCommand(t *testing.T) {
```

Comment:
TBH I'm not sure why we need such a test. This is brittle and doesn't really test any functionality. I'd rather not have it at all.
/cc @crobby

Reply:
I see your point.  Maybe it would be fine to just remove the test and merge as it was.

Reply:
@john801205 Sorry for the back-and-forth. Can you please revert the unit test changes and we'll merge the PR?

Reply:
Sure, no problem. I have reverted the change.

Reply:
Thank you!